### DISCUSSION TITLE
Pooputopia [Chenxi Cui& Hongfan Chen & Liyang Zhang]

### DISCUSSION THREAD

**Discussion by liyang123-z** [realitydeslab/playshop-summer-2024#26] [Proposal]
**Research Questions:**   
我们如何在未来低自由时代创造全新厕所文化，以排便为媒介寻找反叛资本社会规训新出口？
How can we create a new toilet culture in the future era of low freedom, using defecation as a medium to find a new outlet from the social discipline of capital?

**Keywords:**
Poop,Toilet, Acceleration era, awkwardness, 

**Artefact / Media:**
影像+实物 video & a product

**Abstract:**
打造风靡全球的厕所文化：一起去排便吗？
Creating a Global Craze for Toilet Culture：Do we defecate together?

“60年代是人类意识的飞跃。圣雄甘地、马尔科姆X、马丁路德金、切格瓦拉，他们引领了一场良心革命。披头士乐队、大门乐队、吉米亨德里克斯创造了革命和进化主题。音乐就像达利，有很多色彩和革命性的方式。今天的年轻人必须去那里找到自己。”
 —卡洛斯·桑塔纳
"The '60s were a leap in human consciousness. Mahatma Gandhi, Malcolm X, Martin Luther King, Che Guevara, they led a revolution of conscience. The Beatles, The Doors, and Jimi Hendrix created themes of revolution and evolution. Music is like Dali, with many colors and revolutionary ways. Today's young people have to go there and find themselves." -- Carlos Santana

未来2050年，人们的生活充满着规则与监控，舆论自由遭到严重限制，人们生活在一个高监控低自由，高科技低生活的环境，在这种高压环境下，人们的想法得不到阐述发展。粪便是生态循环中重要的一环，排便是无法回避的正常生理需求，排便行为所需的私密性让厕所成为了世界上少有的不被监控的地方。厕所成为了奴役化的恶托邦世界的净土。
（上厕所本身是一个非常轻松的行为——从姿势入手，增加共鸣点，帮助人们构建厕所社区文化）
In the future 2050, people's life is full of rules and monitoring, freedom of public opinion is severely restricted, people live in a high monitoring low freedom, high-tech low life environment, in this high-pressure environment, people's ideas can not be elaborated and developed. Feces are an important part of the ecological cycle, and defecation is an unavoidable normal physiological need. The privacy required for defecation makes the toilet one of the few places in the world that is not monitored. The toilet became the pure land of the enslaved world of distopia.

从不同的视角探讨厕所与厕所、人与人之间的差异与共性，以排便为媒介消解社会观察的严肃性，在其间探究人与厕所的关联，共同创作源于记忆、情感与想象的厕所新文化。至此，排便已经不仅仅是一种行为，更承载了仪式感的象征，深深植根于情感表达之中。从排便开始倾听厕所符号的述说。
Explore the differences and commonalities between toilets and toilets and between people from different perspectives, use defecation as a medium to dissolve the seriousness of social observation, explore the relationship between people and toilets, and jointly create a new toilet culture derived from memory, emotion and imagination. At this point, defecation is not only an act, but also a symbol of ritual, deeply rooted in emotional expression.Listen to the culture of  toilets, starting with the call of nature.

**Visual Reference / Illustrations**
<img width="282" alt="微信图片_20240627150004" src="https://github.com/realitydeslab/playshop-summer-2024/assets/173880451/9e376a4f-0e9b-4a92-8202-3950d16c7ecb">
![微信图片_20240627152528](https://github.com/realitydeslab/playshop-summer-2024/assets/173880451/d243e567-7236-43bd-9a9e-3c23cf380f09)
<img width="663" alt="微信图片_20240627153335" src="https://github.com/realitydeslab/playshop-summer-2024/assets/173880451/72b1fc0e-705e-48c9-b2a2-f4359a28e9bf">

**References**
https://www.gatesfoundation.org/our-work/programs/global-growth-and-opportunity/water-sanitation-and-hygiene/reinvent-the-toilet-challenge-and-expo（bill gates toilet）
https://www.tandfonline.com/doi/pdf/10.1080/15502783.2023.2206380?needAccess=true&role=button

**Comment by liyang123-z** [comment]
在排便乌托邦中，排便姿势的多样性并没有削弱个体之间的共性，而是通过共享的空间使用和心理需求进一步强化了个体之间的同一性。不管姿势如何，人们在上厕所时寻求的放松、隐私和舒适都是一样的。这种共同性使得厕所不仅是一个物理空间，也是一个文化空间，承载着人们对宁静放松的共同期待。In Pooptopia, the diversity of pooping postures does not weaken the commonality between individuals, but further strengthens the identity between individuals through shared space use and psychological needs. Regardless of posture, the relaxation, privacy and comfort that individuals seek when using the toilet are the same. This commonality makes the toilet not only a physical space, but also a cultural space, carrying the common expectation of serenity and relaxation.

**Comment by cynicchen** [comment]
1. 共同体的建立
集体排便作为一种原始而私密的行为，当它被集体化后，便成为了一种特殊的仪式。这种仪式感不仅消解了个人的孤独感，也建立起了一种无言的共同体。这个共同体通过集体行为默默地表达了对现有社会秩序的不满和反抗。
 
2. 身体的解放
在一个强权社会中，身体往往被规训和控制，个人的自由受到各种限制。集体排便这种行为是一种对身体的自然需求的释放，它象征着对身体控制的解放。这种行为在公共场合的表现，实际上是在挑战和嘲弄强权社会对身体的管理和压迫。

 3. 反叛的隐喻
集体排便可以被视为一种对权威和规章制度的嘲弄与挑战。它用一种看似低俗但直接的方式，表达了对强权的蔑视和对自由的追求。这种行为具有一种隐喻的力量，通过最基本的生理需求，传达出深层次的社会反叛意图。

 4. 社会规范的颠覆
强权社会往往依靠严密的社会规范来维持秩序。集体排便这一行为打破了社会对隐私和公共行为的严格界限，颠覆了传统的社会规范。通过这种行为，参与者共同表现出对社会规范的不屑和对自由生活方式的向往。

 5. 无声的抗议
集体排便是一种无声的抗议形式。它不需要语言，不需要标语，却能以一种极具视觉冲击力的方式，直接地传达出对强权的不满和反抗。这种无声的抗议形式，往往更具震撼力，因为它超越了语言的限制，直接诉诸于人们的感官和情感。
 
通过集体排便这一社会行为，可以隐秘而有力地表达对强权社会的反叛。这种行为通过建立共同体、解放身体、颠覆社会规范以及无声抗议等方式，挑战了强权社会的控制和压迫，展现了人们对自由和自主的渴望。


Collective Defecation: A Hidden Rebellion Against Authoritarian Society
Through the social act of collective defecation, we can infer a form of covert rebellion against an authoritarian society. This process can be explored through the following aspects:

1. Establishment of Community
Collective defecation, as a primal and private act, becomes a special ritual when communalized. This ritual not only dissolves the sense of individual loneliness but also establishes a silent community. This community, through collective behavior, silently expresses dissatisfaction and resistance against the current social order.

2. Liberation of the Body
In an authoritarian society, the body is often disciplined and controlled, with personal freedoms restricted. Collective defecation is an act of releasing natural bodily needs, symbolizing the liberation from bodily control. This public display challenges and mocks the authoritarian management and oppression of the body.

3. Metaphor of Rebellion
Collective defecation can be seen as a mockery and challenge to authority and regulations. It uses a seemingly crude but direct method to express contempt for authority and a pursuit of freedom. This act carries metaphorical power, conveying deep social rebellion through basic physiological needs.

4. Subversion of Social Norms
Authoritarian societies rely on strict social norms to maintain order. Collective defecation breaks the strict boundaries between privacy and public behavior, subverting traditional social norms. Through this act, participants show disdain for social norms and a longing for a freer lifestyle.

5. Silent Protest
Collective defecation is a form of silent protest. It requires no words or slogans, yet it conveys dissatisfaction and resistance against authority in a visually impactful way. This silent protest form is often more powerful as it transcends language limitations, directly appealing to people's senses and emotions.

Through the social act of collective defecation, a covert and powerful rebellion against an authoritarian society is expressed. This act challenges authoritarian control and oppression through the establishment of community, bodily liberation, subversion of social norms, and silent protest, showcasing a desire for freedom and autonomy.

**Comment by cynicchen** [comment]
Identity: community, culture, difference
London: Lawrence & Wishart, 1998

**Comment by liyang123-z** [comment]
![微信图片_20240628142956](https://github.com/realitydeslab/playshop-summer-2024/assets/173880451/d11d2929-e13f-4bd4-88b6-a7748a0b756f)
![微信图片_20240628143012](https://github.com/realitydeslab/playshop-summer-2024/assets/173880451/9e7ce9fd-16fd-4a29-a20a-4029b6f3d8db)

**Comment by liyang123-z** [comment]
<img width="481" alt="微信图片_20240628140801" src="https://github.com/realitydeslab/playshop-summer-2024/assets/173880451/8fea6d32-bceb-42b5-be0e-db4e179ba4f5">

**Comment by Axiuuuuu** [comment]
我发现一个新闻！https://www.cbsnews.com/news/parisians-threatening-to-poop-in-the-seine-river-this-weekend-to-protest-the-olympics/

**Comment by glitch-kraft** [comment]
https://en.wikipedia.org/wiki/Fountain_(Duchamp)

**Comment by liyang123-z** [comment]
Keyframe：
![微信图片_20240701210132](https://github.com/realitydeslab/playshop-summer-2024/assets/173880451/021cacb2-53ad-4a51-b73e-76111409cd0b)
Paper pdf ：[论文排版（无引用版）.pdf](https://github.com/user-attachments/files/16061853/default.pdf) incomplete
Tagline： Every poop is a release of the soul; Here, social differences disappear and a sense of collective identity is born. Welcome to Pooputopia.

**Reply by glitch-kraft** [reply]
Whether it's Western or Asian design, toilets are where social differences fade and a collective identity can form. Welcome to Pooputopia.

**Comment by liyang123-z** [comment]
Senario：The person is talking forward, turning to the billboard of Pooutopia, and the person in the billboard turns his head to face the camera

**Comment by glitch-kraft** [comment]
In a near future, where, everything is constantly monitored and recorded, the toilet will remain an island of camera-free individual autonomy. 
However how do we achieve collective autonomy in such future?

**Comment by glitch-kraft** [comment]
In the near future where everything is constantly surveyed and recorded toilet remains an island of camera free autonomy
